### ISSUE TITLE
Blog post image for HA post

### ISSUE THREAD

**Issue by knolleary** [flowforge/website#718]
### Description

**Considerations:**

We have a new blog post to publish regarding how we plan to tackle High Availability within FlowForge.

**Inspirations:**
-  A network with 'packets' coming in from one source and branching out to multiple Node-RED instances

**Comment by Yndira-FlowForge** [comment]
@knolleary is this what you had in mind? I took the idea from [this issue](#649 ) and made it "light" so they wouldn't look too similar. I added the FF logo as the hub that distributes the information. I'm not sure if adding the FF is okay or if the arrows should only go in one direction, so any feedback from you or @joepavitt would be greatly appreciated :)

![Bringing-High-Availability-to-Node-RED](https://github.com/flowforge/website/assets/129537638/913c6579-e0b6-4747-bb1c-a06fcc24c320)
